examples/mcp_servers/authorization/src/authorization/server.py

```diff
@@ -7,44 +7,46 @@
 from arcade_mcp_server import Context, MCPApp
 from arcade_mcp_server.auth import Reddit
 from arcade_mcp_server.resource_server import (
-    AccessTokenValidationOptions,
     AuthorizationServerEntry,
-    ResourceServer,
+    ResourceServerAuth,
 )
 
-# Option 1: Single authorization server example
-resource_server = ResourceServer(
+# Option 1: Single authorization server with custom audience
+# Use expected_audiences when your auth server returns a different aud claim
+# (e.g., client_id instead of canonical_url)
+resource_server_auth = ResourceServerAuth(
     canonical_url="http://127.0.0.1:8000/mcp",
     authorization_servers=[
         AuthorizationServerEntry(  # WorkOS Authkit example configuration
             authorization_server_url="https://your-workos.authkit.app",
             issuer="https://your-workos.authkit.app",
             jwks_uri="https://your-workos.authkit.app/oauth2/jwks",
-            validation_options=AccessTokenValidationOptions(verify_aud=False),
+            expected_audiences=["your-authkit-client-id"],  # Override expected aud claim
```

Comment:
nit: from what we were discussing yesterday it seems like in the WorkOS case it's an app or workspace ID, not a client ID right?

Reply:
So their naming of things is very bad, but it is the AuthKit's Client ID (which is closer to a workspace ID), not the registered client's ID. Both IDs are unfortunately prefixed with `client_` lol